src/routes/rules/components/RulePreview.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <v-row class="mb-10">
-    <v-col lg="4" sm="5">
+    <v-col lg="4" sm="5" cols="12">
```

Comment:
What's the reason for this change? It looks to make the card images too big on small devices:

this branch:
![image](https://github.com/cuttle-cards/cuttle/assets/6520704/9ad0173d-bb91-4cbf-8c89-02729605ae9b)


Vs `main`:
![image](https://github.com/cuttle-cards/cuttle/assets/6520704/337a33a9-1f16-416e-9ac2-2ac7f1a401f2)

Reply:
Here it is in the dialog:
![image](https://github.com/cuttle-cards/cuttle/assets/6520704/eac56563-0faa-4558-ad8a-67a7a74975eb)

Reply:
yes I changed that, I though it looked better on small devices, especially in the modal.. 
is it worth passing a prop to rulepreview for that styling issue ?

Reply:
In this case I think it's preferable to leave it as it was with the cards inline with the text, because then you can read more than one at a time without scrolling. Otherwise it will take twice as much scrolling to read the same amount of text.
```suggestion
    <v-col lg="4" sm="5">
```